### PR TITLE
LEGEND

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

**Comment by jordansread** [comment]
image

File: scripts/visualize/visualize.svg_base_map.R [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
+  checkRequired(depends, 'view-limits')
   
-  # not good to assume order here, but: 
-  view.limits <- depends[[1L]]
-  geoms <- depends[-1L]
-  g.ids <- viz[['depends']][-1L]
+  view.i <- which(names(depends) == 'view-limits')
+  view.limits <- depends[[view.i]]
+  geoms <- depends[-view.i]
+  g.ids <- names(depends)[-view.i]
```
Comment (dblodgett-usgs) [inline scripts/visualize/visualize.svg_base_map.R:11]: 👍 I was trying to change this last night and gave up.

File: viz.yaml [hunk 2]
```diff
@@ -171,7 +193,7 @@ fetch:
     id: nws-data
     location: cache/nws-data.rds
     fetcher: nws_data
-    refetch: FALSE
+    refetch: TRUE
```
Comment (dblodgett-usgs) [inline viz.yaml:196]: 🙏

**Review by dblodgett-usgs — APPROVED** [review]
Looks good. Kinda strange that all my work is shown in here but was already merged? I'm confused.

**Pull request merged**